bitnami/solr/Chart.yaml

```diff
@@ -27,4 +27,4 @@ name: solr
 sources:
   - https://github.com/bitnami/bitnami-docker-solr
   - https://lucene.apache.org/solr/
-version: 2.3.0
+version: 2.3.1
```

Comment:
```suggestion
version: 2.4.0
```

Reply:
I've bumped the minor version for this PR.